frontend/src/pages/SignIn.js

```diff
@@ -20,25 +23,28 @@ const SignIn = ({ onSignIn }) => {
     })
       .then(response => {
         if (response.status === 200)
-          confirmUser(response.json(), history);
+          confirmUser(response.json());
         else if (response.status === 401)
           alert('Incorrent email or password!');
```

Comment:
There's a typo in word `Incorrect`

Reply:
Good, we need to leave the place cleaner than it was